ruby-challenges.rb

```diff
@@ -11,6 +11,21 @@
 # Expected output: '221 is odd'
 
 # Pseudo code:
+# create a function that takes in all variables
+def even_or_odd num
+    # use the modulo operation to determine if the variable does or does not have a remainder which indicates if it is odd or even
+    if num % 2 == 0
+    # return statement if variable is even
+    "#{num} is even"
+    else
+    #  return statement if variable is odd
+    "#{num} is odd"
+    end
+end
+    
+p even_or_odd(reposts1)
+p even_or_odd(reposts2)
+p even_or_odd(reposts3)
```

Comment:
great job!  Don't forget that Ruby has built-in methods that can make this even more efficient!

Reply:
I did a facepalm when I read your comment. Absolutely right I forgot about it.